app/src/test/java/waiter/RouterTest.java

```diff
@@ -7,19 +7,18 @@
 
 public class RouterTest {
 
-    String url, protocol, headers, body;
-    Response.Status status;
+    String url, protocol;
 
 
     @BeforeEach
     void setUp() {
-        url = "URL"; protocol = "PROTOCOL"; headers = "HEADERS"; body = "BODY";
-        status = Response.Status.OK;
+        url = "URL"; protocol = "PROTOCOL";
```

Comment:
protocol = should begin on new line

Reply:
> Looks good. Nice work. Few lint changes, but that's all from me

Thank you for the review Matt!

Reply:
> protocol = should begin on new line

Is this for readability?

Reply:
More syntax-based, `;` usually follows a newline